src/typed/isUndefined.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if the given value is undefined.
+ *
+ * @see https://radashi.js.org/reference/typed/isUndefined
+ * @example
+ * ```ts
+ * isUndefined(undefined) // => true
+ * isUndefined(null) // => false
+ * ```
+ */
+export function isUndefined(value: unknown): value is undefined {
+  return typeof value === 'undefined'
```

Comment:
Count me surprised that this is substantially faster than `=== undefined` (at least on Safari 16 and Chrome 132).

https://www.measurethat.net/benchmarks/Show/15532/0/undefined-vs-typeof-vs

Reply:
:shrug: